ts/src/config.ts

```diff
@@ -115,16 +115,37 @@ export interface Config extends GoogleAuthOptions {
   // For testing with startLocal() only.
   localProfilingPeriodMillis?: number;
 
-
   // Debugging information for startLocal will be recorded every
   // localLogPeriodMillis milliseconds.
   // For testing with startLocal() only.
   localLogPeriodMillis?: number;
 
-
   // Duration of time profiles collected when using startLocal().
   // For testing with startLocal() only.
   localTimeDurationMillis?: number;
+
+  // When true, source map support will be disabled.
+  // All locations in the profiler will reference locations in the running
+  // JavaScript.
+  disableSourceMaps?: boolean;
+
+  // Map of files containing source maps to directories where generated code
+  // refered to by source maps is located. Source maps can be used to
+  // map javascript locations in transpiled code that is running to original
+  // locations in the source code.
+  // All .js.map files which are available in the runtime and within the
+  // application's working directory are added to this map. It is assumed that
+  // these maps will be in the same directory as the generated code.
+  // This can be used to specify source maps which are available in the runtime,
+  // but outside of the working directory, or not in the same directory as the
+  // generated code.
```

Comment:
After reading this a few times, I am still confused about what this does. As a user, what are the scenarios where I would need to specify this config option? How do I know if the default value would just work for my project.

Reply:
+1